src/mlpack/tests/main_tests/isomap_test.cpp

```diff
@@ -0,0 +1,78 @@
+/**
+ * @file isomap_test.cpp
+ * @author Rishabh Ranjan
+ *
+ * Test for Isomap Main.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <string>
+
+#define BINDING_TYPE BINDING_TYPE_TEST
+static const std::string testName = "Isomap";
+
+#include <mlpack/core.hpp>
+#include <mlpack/core/util/mlpack_main.hpp>
+#include "test_helper.hpp"
+#include <mlpack/methods/isomap/isomap_main.cpp>
+
+#include <boost/test/unit_test.hpp>
+
+using namespace mlpack;
+
+struct IsoampFixture
```

Comment:
Misspelling--- `IsoampFixture` -> `IsomapFixture`. :)

Reply:
Next commit would resolve this.